content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins.adoc

```diff
@@ -0,0 +1,1525 @@
+|===
+
+To temporarily disable the proxy configuration, you can add a single
+wildcard character `+*+` on the first line. 
+Erasing the value of *Server* will _discard all values_ and disable 
+the use of a proxy server.
+
+[[JenkinsBehindProxy-Whichpluginssupportaproxyserver]]
+===== Which plugins support a proxy server?
+
+[cols=",,,,",]
+|===
+|*Plugin* |*Since version* |*Supports HTTPS?* |*Supports Exclusions?*
+|*Supports authentication?*
+
+|https://wiki.jenkins.io/display/JENKINS/Fortify+On+Demand+Uploader+Plugin[Fortify
+On Demand Uploader Plugin]
+|https://github.com/jenkinsci/fortify-on-demand-uploader-plugin/releases/tag/fortify-on-demand-uploader-1.08[1.08]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+
+|https://wiki.jenkins.io/display/JENKINS/Git+Plugin[Git Plugin]
+|https://github.com/jenkinsci/git-plugin/releases/tag/git-2.1.0[2.1.0]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+
+|https://wiki.jenkins.io/display/JENKINS/Azure+DevOps+and+Team+Foundation+Server+Plugin[Azure
+DevOps and Team Foundation Server Plugin]
+|https://github.com/jenkinsci/tfs-plugin/releases/tag/tfs-5.1.0[5.1.0]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+
+|https://wiki.jenkins.io/display/JENKINS/Openid+Connect+Authentication+Plugin[Openid
+Connect Authentication Plugin]
+|https://github.com/jenkinsci/oic-auth-plugin/releases/tag/oic-auth-1.4[1.4]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|===
+
+=== Jenkins says my reverse proxy setup is broken
+
+NOTE: Since Jenkins 1.572 this message can also appear if you don't access
+Jenkins through a reverse proxy: Make sure the Jenkins URL configured in
+the System Configuration matches the URL you're using to access Jenkins.
+
+[[Jenkinssaysmyreverseproxysetupisbroken-Symptoms]]
+==== Symptoms
+
+An error message is displayed in the "Manage Jenkins" page - "It appears
+that your reverse proxy set up is broken"
+
+[[Jenkinssaysmyreverseproxysetupisbroken-Background]]
+==== Background
+
+For a reverse proxy to work correctly, it needs to rewrite both the
+request and the response. 
+Request rewriting involves receiving an inbound HTTP call and then making 
+a forwarding request to Jenkins (sometimes with some HTTP headers modified, sometimes not). 
+Failing to configure the request rewriting is easy to catch, because you 
+just won't see any pages at all.
+
+But correct reverse proxying also involves one of two options, EITHER
+
+* rewriting the response (for more information
+see https://wiki.jenkins.io/display/JENKINS/Hyperlinks+in+HTML[Hyperlinks in HTML]). 
```

Comment:
```suggestion
```

Reply:
because this link points to the wiki and it does not add a lot of useful info for the reader